### PR TITLE
Fix typos in release notes.

### PR DESCRIPTION
**Pull request by xpconanfan**
This change is [Reviewable](https://reviewable.io/reviews/google/mobly/914)

### CODE REVIEW

**Review by xianyuanjia — CHANGES_REQUESTED** [review]
> Reviewed 1 of 1 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/google/mobly/914)* status: all files reviewed, 1 unresolved discussion (waiting on @xpconanfan)

___
*[`CHANGELOG.md` line 19 at r1](https://reviewable.io/reviews/google/mobly/914#-NsKwElm2KYIYHJTS_F4:-NsKwElm2KYIYHJTS_F5:bsb0c2v) ([raw file](https://github.com/google/mobly/blob/dfff74ff00f5c666f8b406bd741b6100ddd436e5/CHANGELOG.md#L19)):*
> ```Markdown
> 
> ### Fixes
> * Elinimated redundant `fastboot` calls.
> ```

*Eliminated



_Code quote:_
```Markdown
Elinimated
```

**Review by xpconanfan — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/google/mobly/914)* status: 0 of 1 files reviewed, 1 unresolved discussion (waiting on @xianyuanjia)

___
*[`CHANGELOG.md` line 19 at r1](https://reviewable.io/reviews/google/mobly/914#-NsKwElm2KYIYHJTS_F4:-NsKwZGS7voIq4xmsyCd:b-896fix) ([raw file](https://github.com/google/mobly/blob/dfff74ff00f5c666f8b406bd741b6100ddd436e5/CHANGELOG.md#L19)):*
<details><summary><i>Previously, xianyuanjia wrote…</i></summary><blockquote>

*Eliminated
</blockquote></details>

Done.

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,14 +9,14 @@ Removed legacy code and dependencies.
 ### New
 * Support am instrument options in snippet client.
 * Support adb reverse in `AdbProxy`.
-* Improved mechaism for tracking repeat and retry records in test report.
+* Improved mechanism for tracking repeat and retry records in test report.
 
 ### Breaking Changes
```
Comment (mhaoli) [inline CHANGELOG.md:14]: Should we mention python min version bump?

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,14 +9,14 @@ Removed legacy code and dependencies.
 ### New
```
Comment (mhaoli) [inline CHANGELOG.md:9]: Does it worth mentioning the support of configuring snippet user in #899 ?

File: CHANGELOG.md (hunk 1 above)
Comment (xpconanfan) [reply CHANGELOG.md:14]: Isn't it already mentioned?

File: CHANGELOG.md (hunk 2 above)
Comment (xpconanfan) [reply CHANGELOG.md:9]: I think it's fine.

**Review by mhaoli — APPROVED** [review]

**Review by xianyuanjia — APPROVED** [review]
:lgtm:

> Reviewed 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/google/mobly/914#-:-NsPjizJ61mNl7db9gVo:bnfp4nl)* status: all files reviewed, 3 unresolved discussions (waiting on @mhaoli)

**Pull request merged**